core/src/test/java/com/google/instrumentation/stats/DistributionAggregationTest.java

```diff
@@ -28,16 +28,19 @@
  */
 @RunWith(JUnit4.class)
 public final class DistributionAggregationTest {
+
+  private static final double TOLERANCE = 1e-6;
+
   @Test
   public void testDistributionAggregationWithOutBuckets() {
     DistributionAggregation aggr = DistributionAggregation.create(10, 5.0, 30.0,
         Range.create(1.0, 5.0), TAGS);
 
     assertThat(aggr.getCount()).isEqualTo(10);
-    assertThat(aggr.getMean()).isEqualTo(5.0);
-    assertThat(aggr.getSum()).isEqualTo(30.0);
-    assertThat(aggr.getRange().getMin()).isEqualTo(1.0);
-    assertThat(aggr.getRange().getMax()).isEqualTo(5.0);
+    assertThat(aggr.getMean()).isWithin(TOLERANCE).of(5.0);
+    assertThat(aggr.getSum()).isWithin(TOLERANCE).of(30.0);
+    assertThat(aggr.getRange().getMin()).isWithin(TOLERANCE).of(1.0);
+    assertThat(aggr.getRange().getMax()).isWithin(TOLERANCE).of(5.0);
```

Comment:
This test is just checking that the DistributionAggregation's fields are set correctly and isn't doing any double math, so I think the exact comparison is better.